01-algorithms/02-algorithms-divide-conquer.md

```diff
@@ -1,42 +1,24 @@
 # Divide and Conquer
 
-<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=04387545-7ced-4c84-9ce4-ad44002bde36&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
-
 ## Goals
 
-As developers, we need as many tools to solve problems as possible.
-
-When we understand the _kind_ of solution or algorithm we are using or can use, we can more easily read and understand it.
-
-There are a number of solutions and algorithms classified as using the divide-and-conquer approach. Learning about divide-and-conquer approaches will:
+You may recall learning about divide and conquer algorithms such as Binary Search, Quick Sort, and Merge Sort. This lesson is intended to serve as a refresher for the divide and conquer approach.
```

Comment:
Move binary search to the end since it's _techically_ not divide and conquer.
```suggestion
You may recall learning about divide and conquer algorithms such as Quick Sort, and Merge Sort, Binary Search. This lesson is intended to serve as a refresher for the divide and conquer approach.
```